sagemaker-neo-notebooks/edge/cpp-integration/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# CMakeLists.txt
+cmake_minimum_required(VERSION 3.2)
+
+option(BUILD_SHARED_LIBS "Build shared libraries" ON)
+
+project(tutorial)
+
+# for debugging only
+# set(CMAKE_CXX_FLAGS "-g")
```

Comment:
cmake has a flag to enable debug mode, something like the following can be used:
```
if ("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
    set(CMAKE_CXX_FLAGS "-g")
endif()
```

Reply:
it's revised as suggested.